.github/workflows/test.yml

```diff
@@ -81,9 +81,9 @@ jobs:
       run: npm run build
     - name: Test experimental features
       env:
-        ASC_FEATURES: threads,reference-types,gc,exception-handling
+        ASC_FEATURES: "*"
       run: |
-        npm run test:compiler features/threads features/reference-types features/gc features/exception-handling
+        npm run test:compiler
```

Comment:
Does it repeat to run lots of test which has already done in `npm test`?

Reply:
Sadly, yes. Maybe this can be changed to somehow run only tests requiring `ASC_FEATURES=whatever`.